README.md

```diff
@@ -208,6 +208,10 @@ For example, if `A isa Base.Matrix`, `offsets(A) === (StaticInt(1), StaticInt(1)
 
 Is the function `f` whitelisted for `LoopVectorization.@avx`?
 
+## static(x)
```

Comment:
Are we at the point yet where we should just create a `Static.jl` and depend on it?

We have static `Int`s, `Bool`s, and now `Symbol`s.
Octavian also has [staticfloats](https://github.com/JuliaLinearAlgebra/Octavian.jl/blob/master/src/staticfloats.jl).

Reply:
I was thinking the same thing. There is already StaticNumbers.jl but I think there is a lot of utility in a _very_ restricted set of static types. I think there has been a tendency with projects that deal with "staticness" to just solve everything with a new static type, which creates another problem.

Reply:
Where do think a package like `Static.jl` should go?

Reply:
I just created [SciML/Static.jl](https://github.com/SciML/Static.jl) and added you as a contributor.

I'm not sure about ArrayInterface's functionality vs StaticNumbers.jl, but ArrayInterface's has definitely been growing.
I should've added static ranges to the list of things we support here.

But I do agree on the idea of having a narrower scope / focusing on being a lightweight dependency that other packages that want such functionality (including of course ArrayInterface.jl) can cheaply depend upon.